### PR TITLE
chore: docs/usage update

### LINKED ISSUE #180: Command errors on usage.md
[docs/usage](https://github.com/openshift/oc-mirror/blob/main/docs/usage.md) has example commands using the command `oc-bundle` should this be replaced with `oc-mirror`?

A minor nitpick on the same page, would it be better to use the generic pull secret link `https://console.redhat.com/openshift/install/pull-secret` instead of `https://console.redhat.com/openshift/install/aws/installer-provisioned`?

### PR DESCRIPTION
**Pull request by jpower432**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #180 

## Type of change

- [X] This change requires a documentation update

# How Has This Been Tested?

N/A

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -1,55 +1,120 @@
...
+### Running `oc-mirror` For Differential Updates
 
-`oc-bundle publish --dir=<< Empty Directory >> --archive=bundle_000000.tar --to-mirror=<< fqdn:port of target mirror >>`
+Once a full imageset has been created and published, differential imagesets that contain only updated images as per the configuration file can be generated with the same command as above:
 
-To publish a differential imageset use the following command:
+`./bin/oc-mirror --config imageset-config.yaml --dir test-create file://archives`
 
-`oc-bundle publish --dir=<< previously used dir >> --archive=bundle_000000.tar --to-mirror=<< fqdn:port of target mirror >>`
+**Note 1:** Depending on the configuration file used and the periodicity between running `oc-mirror`, this process may download multiple-hundreds of gigabytes of data, though differential updates should usually result in a significantly smaller imagesets.
```
Comment (estroz) [inline docs/usage.md:115]: We should probably hoist this note as a warning to the top of this doc, and generalize it to say "this tool might downloaded many gigabytes of data to your local disk. ensure you have ~100GB free space before running the tool".

**Review by estroz — CHANGES_REQUESTED** [review]

File: docs/usage.md [hunk 2]
```diff
@@ -1,55 +1,120 @@
...
 
 **Notice:** These commands are early alpha and may change significantly between application versions. 
 
-oc-bundle has two main modes of operation: `create` and `publish`. 
-
 ## Prerequisites
 
 ### Authentication: 
-oc-bundle currently retrieves registry credentials from `~/.docker/config.json`. Make sure that your [Red Hat OpenShift Pull Secret](https://console.redhat.com/openshift/install/aws/installer-provisioned) and any other needed registry credentials are populated in `~/.docker/config.json`
+oc-mirror currently retrieves registry credentials from `~/.docker/config.json`. Make sure that your [Red Hat OpenShift Pull Secret](https://console.redhat.com/openshift/install/pull-secret) and any other needed registry credentials are populated in `~/.docker/config.json`
```
[the remark this answers is not in the document]
Comment (jpower432) [reply docs/usage.md:27]: I think it is worth adding a blurb in a follow on PR about what the differences in the command will look like.

File: README.md [hunk 3]
```diff
@@ -80,6 +84,10 @@ Replace this value with a real registry host, or create a `docker.io/library/reg
     ```sh
     ./bin/oc-mirror describe /path/to/archives
```
Comment (estroz) [inline README.md:85]: Just a few more of these kicking around
```suggestion
    oc-mirror describe /path/to/archives
```

Comment (estroz) [reply README.md:85]: Whoops this is supposed to be here.

**Review by estroz — APPROVED** [review]
/lgtm

**Pull request merged**